rackcity/models/network_port.py

```diff
@@ -25,6 +26,46 @@ class NetworkPort(models.Model):
         blank=True,
     )
 
+    def create_network_connection(self, destination_port):
```

Comment:
per our slack conversation, perhaps rename `network_connection` field to `connected_port` so it's obvious that this entity is another NetworkPort, and not some sort of connection object

Reply:
I like this - changed to reflect